Mathlib/Combinatorics/Additive/VerySmallDoubling.lean

```diff
@@ -362,7 +362,10 @@ omit [DecidableEq G] in
 private lemma op_smul_eq_op_smul_iff_mem {H : Subgroup G} {x y : G} :
     x ∈ (H : Set G) <• y ↔ (H : Set G) <• x = H <• y := op_smul_eq_iff_mem (mem_orbit _ _)
 
-def convolution (A B : Finset G) : G → ℕ := fun x => #{ab ∈ A ×ˢ B | ab.1 * ab.2⁻¹ = x}
+/-- Given finite subsets `A` and `B` of a group `G`, convolution of `A` and `B` is a map `G → ℕ`
+that for a given `x ∈ G` returns the number of representations of `x` in the form `x = ab⁻¹`, where
+`a ∈ A`, `b ∈ B`. -/
+private def convolution (A B : Finset G) : G → ℕ := fun x => #{ab ∈ A ×ˢ B | ab.1 * ab.2⁻¹ = x}
```

Comment:
This should not be private and be put in a new file `Combinatorics.Additive.Convolution` (or something like that)